WebDriverAgentLib/Routing/FBResponsePayload.m

```diff
@@ -89,9 +92,35 @@
   NSMutableDictionary *dictionary = [NSMutableDictionary new];
   dictionary[@"ELEMENT"] = elementUUID;
   if (!compact) {
-    XCElementSnapshot *snapshot = element.fb_lastSnapshot;
-    dictionary[@"type"] = snapshot.wdType;
-    dictionary[@"label"] = snapshot.wdLabel ?: [NSNull null];
+    NSArray *fields = [FBConfiguration.elementResponseAttributes componentsSeparatedByString:@","];
+    XCElementSnapshot *snapshot = element.fb_lastSnapshotFromQuery;
+    for(NSString *field in fields) {
+      if ([field isEqualToString:@"name"] || [field isEqualToString:@"type"]) {
```

Comment:
name and type are not the same values

Reply:
we need to keep the backward compatibility. in WDA `type` is the element type, for example `XCUIElementTypeCell` and name is the accessibility id of the element.

Reply:
But then in appium-xcuitest-driver, we would need to translate the names when setting the elementResponseFields capability/setting; and also translate getElement(s) responses attribute names back to W3C.

Reply:
such a pain...
My point is that the attribute names we get via getAttribute and these should be in sync. Can you provide a link to the spec?

Reply:
I used this https://www.w3.org/TR/webdriver/#list-of-endpoints - see the .../element/xxx endpoints

Reply:
so then there should be no conflict. name can be matched to wdName and type to wdType - this is not in conflict with what I see there.

Reply:
But element/<id>/name is matched to wdType, not wdName
https://github.com/facebook/WebDriverAgent/blob/f81e120cdf6f74185f0f653c23171e5c63ee5d74/WebDriverAgentLib/Commands/FBElementCommands.m#L141

Reply:
ok, I got it. A small comment in the code might also be useful